py/client/tests/test_sync.py

```diff
@@ -0,0 +1,38 @@
+
+
+import unittest
+from time import sleep
+
+import pyarrow as pa
+from pyarrow import csv
+
+from pydeephaven import DHError
+from pydeephaven import Session
+from tests.testbase import BaseTestCase
+
+
+class SyncTestCase(BaseTestCase):
+    def test_persistent_tables(self):
+        with Session() as session1:
+            session1 = Session()
+            session1.run_script('t = None')
+            t = session1.empty_table(10)
+            session1.bind_table('t', t)
+
+        sleep(1)
```

Comment:
Do we need this sleep? I think we can probably formulate these tests w/o needing to rely on them.

Reply:
Anytime I see a sleep, it immediately triggers 'code smell'. I wonder if we should have a way for session1 to block on the successful notification that `t` has been propogated.

Reply:
Fixed by using a loop and a timeout instead (though the test is skipped for other reasons anyways...)